### PR TITLE
Carpet Protocol Support

### PR DESCRIPTION
**Pull request by X-Green**
CreativeNoClip Available;
SmoothClientAnimation not available yet, no idea what went wrong...

### CODE REVIEW

File: genPatches.bat [hunk 1]
```diff
@@ -0,0 +1 @@
+gradlew genpatches
```
Comment (Fallen-Breath) [inline genPatches.bat]: 这个文件爪巴

File: runServer.bat [hunk 2]
```diff
@@ -0,0 +1 @@
+D:\Documents\Github\TIS-Carpet\gradlew runServer
```
Comment (Fallen-Breath) [inline runServer.bat:1]: 这个文件爪巴

**Comment by Fallen-Breath** [comment]
stop being buggy
go fix unexpected behavior
and test more ofc

**Comment by X-Green** [comment]
orz

**Comment by Fallen-Breath** [comment]
838fa3e74dddbc51fbbb81e8f0f03a20bc90fbd7 ~ 36a0491ff1a13fb8e5f4773e766c0fed0f11ba21

**Pull request closed without merging**